spec/views/splash/welcome_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+RSpec.describe Category, type: :system do
+  describe 'splash page' do
+    before :each do
+      visit root_path
+    end
+
+    it 'display hello' do
+      expect(page).to have_content 'Hello!'
+    end
+
+    it 'display app name' do
+      expect(page).to have_content 'Exchange budget'
+    end
+
+    it 'display hello' do
+      click_on 'Sign in'
+      expect(page).to have_content 'Log in'
+    end
+
+    it 'display hello' do
+      click_on 'Sign up'
+      expect(page).to have_content 'Sign up'
+    end
+  end
```

Comment:
- [ ] For the project requirements, Kindly ensure that all your tests are passing as right there is one test case that is failing, you can find the screenshot for the result below:
![Screenshot from 2023-05-31 14-32-43](https://github.com/Tobby8629/Budget_app/assets/83042742/f7092a10-c9b9-400f-a054-d3499e39244c)